ext/drwrap/drwrap.c

```diff
@@ -946,9 +951,11 @@ drwrap_init(void)
     drmgr_init();
     if (!drmgr_register_bb_app2app_event(drwrap_event_bb_app2app, &pri_replace))
```

Comment:
I see that "replace" is performed by `drwrap_event_bb_app2app`. So, the fact that we don't conditionally register `drwrap_event_bb_app2app` means that drwrap will perform "replace" even if `DRWRAP_INVERT_CONTROL` is set. So, control is inverted only for "wrap" but not "replace"? That can be confusing.

How about we drop support for "replace" if `DRWRAP_INVERT_CONTROL` is set? It seems less likely that the user will want such mix-and-match anyway.

Reply:
Done.